core/session.ts

```diff
@@ -444,6 +431,21 @@ export class Session {
     return encode64(dataform.CompiledGraph, this.compile());
   }
 
+  public normalizeDatabase(database: string): string {
+    return this.adapter().normalizeIdentifier(
+      `${database}${this.getDatabaseSuffixWithUnderscore()}`);
+  }
+
+  public normalizeSchema(schema: string): string {
+    return this.adapter().normalizeIdentifier(
+      `${schema}${this.getSchemaSuffixWithUnderscore()}`);
+  }
+
+  public normalizeTable(table: string): string {
```

Comment:
this should be `normalizeName`. i realise that "name" as a term really sucks, but it is what the entire framework (and other code which uses the framework) uses to indicate the last part of a target, which could be a table, view, or any other materializable object - or might not even be materializable, in the case of `operations`!

Reply:
Ack. Have updated alongside the `normalize` => `finalize` change.